docs/reference/README.md

```diff
@@ -330,31 +246,43 @@ The wallets below support FIL tokens. We recommend verifying that these have bee
-  - Also available on npm at [@blitslabs/filecoin-js-signer](https://www.npmjs.com/package/@blitslabs/filecoin-js-signer)
-  - Currently used by the Filecoin Loans project for its mobile wallet (written in React Native) and creating Payment Channels.
+### Storage apps on Filecoin
 
-### Retrieval Market resources (WIP experiments)
+- [**Slate.host**](https://slate.host) - - **_recommended_**
+  - a storage application on Filecoin to collect, organize, and link files together and share them, listed on [Product Hunt here](https://www.producthunt.com/posts/slate-f195dcdd-18e2-4dc2-8c70-45208ccbb862) on Github at [filecoin-project/slate](https://github.com/filecoin-project/slate/)
+- [Starling Storage API + CLI](https://github.com/filecoin-project/starling) - RESTful API + Node.js CLI that simplifies storing data for preservation use cases. Visit [Starlingstorage.io](https://starlingstorage.io/) to learn more.
+- [Starling Framework for Data Integrity](https://www.starlinglab.org/)
+  - to securely capture, store and verify human history
+  - learn more at [starlinglab.org/78days](https://www.starlinglab.org/78days/) or the [Filecoin blog interview](https://filecoin.io/blog/starling-framework/)
+- [ChainSafe Files](https://files.chainsafe.io/) - Dropbox-style UI, login with Oauth or general Metamask
+- [File.video](https://file.video/) and [Voodfy](https://beta.voodfy.com/) - video hosting with decentralized transcoding from LivePeer
 
-The Retrieval Market in Filecoin is currently a basic version and research and development teams are actively exploring enhancements.
+### Grant programs and accelerators
 
-- [Decentralized Data Delivery Markets - Open Problems and RFCs](https://github.com/protocol/ResNetLab/blob/master/OPEN_PROBLEMS/DECENTRALIZED_DATA_DELIVERY_MARKETS.md)
-- [ChainSafe's Full Node Retrieval Client](https://github.com/ChainSafe/fil-secondary-retrieval-markets)
-- [Browser Retrieval Client](https://github.com/filecoin-shipyard/browser-retrieval)
-- [Browser Retrieval Client using WASM](https://github.com/jimpick/lotus-retrieve-api-daemon)
-- [July 2020 Retrieval Markets Workshop](https://www.youtube.com/watch?v=eUcsZ1JS9pM) - list of [Sessions](https://docs.google.com/document/d/17bEHP2CHkQFYYQnl7YpatAOh6kWxjlNgihdc4F66SVM/edit)
+- [**Filecoin Dev Grants**](https://filecoin.io/grants) - ongoing quarterly dev grant program to support open source projects + new [RFPs](https://github.com/filecoin-project/devgrants/tree/master/rfps) on Filecoin
+  - funded by the [Filecoin Foundation](https://fil.org)
+- [Upcoming Hackathons](https://github.com/protocol/grants)
+  - Keep a lookout for upcoming hackathons! All winners are eligbile for [Next Steps Grants](https://github.com/filecoin-project/devgrants/blob/master/microgrants/microgrants.md) after the hackathon
+- Past Hackathons
```

Comment:
Do you think it is a good idea to list hackathons here since like you said it will get out of date easily? Because it is a reference page, we should just have a link to the page or github where all the hackathons are collected (if we have one, maybe [here](https://github.com/protocol/grants#hackathons)?)

Reply:
Yeah I'd suggest linking to the `procotol/grants` page to stop this section becoming out of date.